profiles/wellingtonmwadali.json

```
@@ -0,0 +1,7 @@
+{
+    "image": "https://github.com/wellingtonmwadali",
+    "issueId": 1675,
+    "name": "Wellington Mwadali",
+    "username": "wellingtonmwadali"
+  }
+  
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```